streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopicManagerTest.java

```diff
@@ -64,6 +64,7 @@
 import static org.apache.kafka.common.utils.Utils.mkEntry;
 import static org.apache.kafka.common.utils.Utils.mkMap;
 import static org.apache.kafka.common.utils.Utils.mkSet;
+import static org.easymock.EasyMock.expect;
```

Comment:
Could you please revert this change since I am still working on this code for KIP-698 and I do not want to run into merge conflicts.